### ISSUE TITLE
[FEATURE] MLflow tests

### ISSUE THREAD

**Issue by mr-sarthakgupta** [odysseyiitr/Sklearn-genetic-opt#1]
Problem: Currently there are no unit tests for integration with MLflow

Difficulty Level - Hard

Steps:
- Create the file in sklearn_genetic/tests/test_mlflow.py and put the set of test that contains the use case of MLflow from sklearn_genetic.mlflow
- It should test if the config creates or no a new topic and the use of each parameter, as well, that at the end of the runs the logged artifacts/metric/hyperparameters exists in the mlflow server and clean the resources after the test is ended
